snuba/migrations/snuba_migrations/events/0010_groupedmessages_onpremise_compatibility.py

```diff
@@ -0,0 +1,91 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.cluster import ClickhouseClientSettings, get_cluster
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+TABLE_NAME = "groupedmessage_local"
+TABLE_NAME_NEW = "groupedmessage_local_new"
+TABLE_NAME_OLD = "groupedmessage_local_old"
+
+
+def fix_order_by() -> None:
+    cluster = get_cluster(StorageSetKey.EVENTS)
+    clickhouse = cluster.get_query_connection(ClickhouseClientSettings.MIGRATE)
+    database = cluster.get_database()
+
+    new_primary_key = "project_id, id"
+    old_primary_key = "id"
+
+    ((curr_primary_key,),) = clickhouse.execute(
+        f"SELECT primary_key FROM system.tables WHERE name = '{TABLE_NAME}' AND database = '{database}'"
+    )
+
+    assert curr_primary_key in [new_primary_key, old_primary_key]
```

Comment:
please add an error message explaining why their migration failed. This exception will bubble up to the user.